src/metrics/perf_event.rs

```diff
@@ -59,6 +59,52 @@ pub struct PerfEvents {
     pub counters: Vec<HardwareCounter>,
 }
 
+/// Configurable hardware counters that may be added to an Operator's config
+///
+///```no_run
+/// use arcon::prelude::*;
+/// #[derive(Default)]
+/// pub struct MyOperator;
```

Comment:
Good idea to include a simple application. We can however use an already defined operator.

```rust
operator: Arc::new(|| Map::new(|x| x + 10))
```

Also, please make sure it's indented properly.